include/openmc/nuclide.h

```diff
@@ -197,7 +197,12 @@ class Nuclide {
 
           // Randomly sample between temperature i and i+1
           f = (kT - kTs_[i_temp]) / (kTs_[i_temp + 1] - kTs_[i_temp]);
-          if (f > prn(p.current_seed())) ++i_temp;
+
+          //TODO: to maintain the same random number stream as the Fortran code this
+          //replaces, the seed is set with index_ + 1 instead of index_
+          double sample = future_prn(static_cast<int64_t>(index_ + 1), p.seeds_[STREAM_SAB_T]);
```

Comment:
Based on that comment, I think we can just change to `index_` since we're not trying to preserve reproducibility with previous code versions

Reply:
Agreed!